Cartfile

```diff
@@ -1,4 +1,4 @@
-binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" == 23.0.0-rc.2
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" ~> 114.0
-github "mapbox/mapbox-directions-swift" == 2.7.0-beta.1
+binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" == 23.0.0
+binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" == 115.0
```

Comment:
I don't think that we need to pin to `v115.0` of MapboxNavigationNative.

Reply:
Thanks for the suggestion, I removed the pining exact version to MapboxNavigationNative and MapboxMaps

Reply:
Thank you, there is one comment regarding Maps SDK version resolving, when it's changed I think that we're good to go.